cypress/integration/1221-my_publication/354-creator_opens_my_publications.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Open My Publications

```

Comment:
Hva er egentlig forskjellen på `Feature` og `Scenario`?

Reply:
En `Feature` kan inneholde flere `Scenario`. Siden vi splittet opp mvp.feature har vi kun ett `Scenario` pr feature-fil. Det finnes også `Scenaroi Outline` hvor du legger ved en tabell i `Examples` for å kjøre et `Scenario` med flere forskjellige verdier.

Reply:
En Feature kan ha flere Scenarios. Men vi har egentlig ikke brukt det sånn i våre prosjekt. 

F.eks: 
Feature: Open my publications
Scenario: Open my publications succeeds
Scenario: Open my publications fails